iphone/Maps/Classes/CarPlay/CarPlayWindowScaleAdjuster.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import UIKit
+
+@objc
+class CarPlayWindowScaleAdjuster: NSObject {
+
+  @objc
+  static func updateAppearance(
```

Comment:
Can't we call this code from `setup()`/`destroy()`? Then we don't need this separate class.

Reply:
Good point 👍  Moved this call to `setup()`/`destroy()`, but left `CarPlayWindowScaleAdjuster` enum (namespace) to not overcomplicate `CarPlayService`